### DISCUSSION TITLE
Find and run shell scripts in project directory

### DISCUSSION THREAD

**Discussion by yngwi** [stevearc/overseer.nvim#42] [Q&A] [answered]
Hi! Thank you for helping me so quickly last time. I have another noob question: is it possible to have overseer look for shell scripts in the current project directory and dynamically add a launch task to `OverseerRun` for each of them?

**Comment by stevearc** [comment, marked as answer]
I've added this as a recipe in the documentation:
https://github.com/stevearc/overseer.nvim/blob/master/doc/recipes.md#run-shell-scripts-in-the-current-directory

See the guide on [Custom tasks](https://github.com/stevearc/overseer.nvim/blob/master/doc/guides.md#custom-tasks) for information on how to register that template so the tasks show up in `:OverseerRun`

**Reply by yngwi** [reply]
Great, thank you!
